frontend/src/container/WorkspaceContent.jsx

```diff
@@ -110,7 +110,16 @@ class WorkspaceContent extends React.Component {
       else return
     } else wsToLoad = match.params.idws
 
-    this.loadContentList(wsToLoad)
+    await this.loadContentList(wsToLoad)
+
+    let contentToJump = this.getContentIdOpenedInUrl(match.params)
```

Comment:
put the code related to the scroll in a specific function and rename variable `contentToJump` to `contentToScrollTo` please